api/agent/drivers/docker/registry_test.go

```diff
@@ -57,3 +60,54 @@ func TestRegistrySubDomains(t *testing.T) {
 		t.Fatalf("subdomain results failed expected[%+v] != results[%+v]", exp, res)
 	}
 }
+
+func TestRegistryEnv(t *testing.T) {
+
+	testCfg := `{
+	"auths":{
+		"https://my.registry.com":{"auth":"Y29jbzpjaGVlc2UK"},
+		"https://my.registry.com:5000":{"auth":"Y29jbzpjaGVlc2UK"},
```

Comment:
No email and password style auths needed?

Reply:
Yes, not testing those... We pass that directly to fsouza/docker currently...